lint-tasks.yml

```diff
@@ -31,13 +42,29 @@ tasks:
         vars:
           FLAGS: "--dry-run"
 
-  cpp-fix:
-    sources: *cpp_source_files
+  cpp-format-fix:
+    sources: *cpp_format_source_files
     cmds:
       - task: "clang-format"
         vars:
           FLAGS: "-i"
 
+  cpp-static-check:
+    # Alias task to `cpp-static-fix` since we don't currently support automatic fixes.
+    # NOTE: clang-tidy does have the ability to fix some errors, but the fixes can be inaccurate.
+    # When we eventually determine which errors can be safely fixed, we'll allow clang-tidy to
+    # fix them.
+    aliases: ["cpp-static-fix"]
+    sources:
```

Comment:
Do we need to add `G_EMSDK_CHECKSUM` here?

Reply:
Yeah, my mistake. Let me restructure things to be clearer.

Reply:
Since we plan to move clang-format and clang-tidy into yscope-dev-utils, what do you think about the following? Move clang-format and clang-tidy's dependencies into cpp-format-* and cpp-static-* and make clang-format and clang-tidy idempotent.